public/components/overview/mitre/components/techniques/techniques.tsx

```diff
@@ -281,6 +303,12 @@ export const Techniques = withWindowSize(class Techniques extends Component {
     this.props.onSelectedTabChanged('dashboard');
   }
 
+  openIntelligence(e,redirectTo,itemId){
+    sessionStorage.setItem("tabRedirect", redirectTo);
+    sessionStorage.setItem("idToRedirect", itemId);
+    this.props.onSelectedTabChanged('intelligence');
+  }
+
```

Comment:
Maybe we could do the redirection through a query param in the URL instead of using the `sessionStorage`.

Reply:
Yes, it was my mistake. I fix this